posthog/hogql_queries/query_runner.py

```diff
@@ -429,15 +432,26 @@ def apply_dashboard_filters(self, dashboard_filter: DashboardFilter) -> Q:
             query_update: dict[str, Any] = {}
             if dashboard_filter.properties:
                 if self.query.properties:
-                    query_update["properties"] = PropertyGroupFilter(
-                        type=FilterLogicalOperator.AND,
-                        values=[
-                            PropertyGroupFilterValue(type=FilterLogicalOperator.AND, values=self.query.properties),
-                            PropertyGroupFilterValue(
-                                type=FilterLogicalOperator.AND, values=dashboard_filter.properties
-                            ),
-                        ],
-                    )
+                    try:
+                        query_update["properties"] = PropertyGroupFilter(
+                            type=FilterLogicalOperator.AND,
+                            values=[
+                                PropertyGroupFilterValue(type=FilterLogicalOperator.AND, values=self.query.properties)
+                                if isinstance(self.query.properties, list)
+                                else PropertyGroupFilterValue(**self.query.properties.model_dump()),
+                                PropertyGroupFilterValue(
+                                    type=FilterLogicalOperator.AND, values=dashboard_filter.properties
+                                ),
+                            ],
+                        )
+                    except Exception as e:
+                        # If pydantic is unhappy about the shape of data, let's ignore property filters and carry on
+                        capture_exception()
+                        logger.error(
+                            "Failed to apply dashboard property filters",
+                            exception=e,
+                            exc_info=True,
```

Comment:
Can also use `logger.exception` and then you don't need the `exc_info=True`

Reply:
Oh yeah, that's exactly what I was looking for